modules/administration/pages/custom-ssl.adoc

```diff
@@ -1,97 +1,195 @@
+rhn-ssl-tool --gen-server --dir="/root/ssl-build" --set-country="COUNTRY" \
+--set-state="STATE" --set-city="CITY" --set-org="ORGANIZATION" \
+--set-org-unit="ORGANIZATION UNIT" --set-email="name@example.com" \
+--set-hostname="susemanager.example.top" --set-cname="example.com"
 ----
-. Execute {productname} setup with
+Ensure that the [systemitem]``set-cname`` parameter is the fully-qualified domain name of your {productname} Server.
+You can use the the [systemitem]``set-cname`` parameter multiple times if you require multiple aliases.
+. Install the RPM that contains the newly generated certificate:
 +
-
 ----
-yast2 susemanagersetup
+rpm -Uhv /root/ssl-build/lnx0259a/rhn-org-httpd-ssl-key-pair-lnx0259a-1.0-2.noarch.rpm
 ----
+. Restart services to pick up the changes:
 +
-Proceed with the default setup.
-Upon reaching the Certificate Setup window during {yast}
-installation, fill in random values, as these will be overridden with the values specified in <<bp.cert.custom.setup.proc.export>>.
-+
-.Shell Requirements
-NOTE: Make sure that you execute [command]``yast2
-      susemanagersetup`` from within the same shell the environment variables were exported from.
-+
+----
+spacewalk-service restart
+----
 
 
 
-[[bp.cert.custom.proxy]]
-== Using a Custom Certificate with {susemgrproxy}
+== Create New Certificates
 
+If you need to create entirely new certificates for an existing installation, you will need to delete the old certificates before you begin.
 
-After completing the installation with yast found in <<advanced.topics.proxy.quickstart>> continue with a modified <<at.manager.proxy.run.confproxy>> procedure:
+.Procedure: Creating New Certificates
 
-[[bp.cert.custom.proxy.proc]]
+. On the {productname} Server, at the command prompt, move the old certificate directory to a new location:
++
+----
+mv /root/ssl-build /root/old-ssl-build
+----
+. Generate a new CA certificate:
++
+----
+rhn-ssl-tool --gen-ca --dir="/root/ssl-build" --set-country="COUNTRY" \
+--set-state="STATE" --set-city="CITY" --set-org="ORGANIZATION" \
+--set-org-unit="ORGANIZATION UNIT" --set-common-name="SUSE Manager CA Certificate" \
+--set-email="name@example.com"
+----
+. Generate a new server certificate:
++
+----
+rhn-ssl-tool --gen-server --dir="/root/ssl-build" --set-country="COUNTRY" \
+--set-state="STATE" --set-city="CITY" --set-org="ORGANIZATION" \
+--set-org-unit="ORGANIZATION UNIT" --set-email="name@example.com" \
+--set-hostname="susemanager.example.top" --set-cname="example.com"
+----
+Ensure that the [systemitem]``set-cname`` parameter is the fully-qualified domain name of your {productname} Server.
+You can use the the [systemitem]``set-cname`` parameter multiple times if you require multiple aliases.
+. Generate the CA certificate RPM:
++
+----
+rhn-ssl-tool --gen-ca --rpm-only --dir="/root/ssl-build" \
+--from-ca-cert=`path_to_certificate_file`
```

Comment:
I think this is not needed here. The commands above should automatically generate the RPMs.